fireworks/tests/mongo_tests.py

```diff
@@ -597,5 +607,44 @@ def test_stats(self):
         self.assertEqual((workflow_results[0]["_id"], workflow_results[0]["count"]), ("COMPLETED", 3))
 
 
+class SQLiteTests(MongoTests):
```

Comment:
@shyamd yeah I'm going through the existing mongo tests with this class, one by one.  unittest is pretty awful, the cleanup doesn't always seem to work so once one test fails they all do...